tests/integration/test_records.py

```diff
@@ -425,3 +425,23 @@ def test_responses(client, simple_ensemble):
     assert len(responses) == 1
     data = client.get(f"/records/{responses['rec3']['id']}/data").json()
     assert data == [[1, 2], [4, 5]]
+
+
+def test_ensemble_wide_parameters(client, simple_ensemble):
```

Comment:
Not sure how this test reflects the metadata

Reply:
Ooops. I accidentally staged code I shouldn't 🤣